torch/distributions/multivariate_normal.py

```diff
@@ -165,7 +166,7 @@ def __init__(
                 covariance_matrix.shape[:-2], loc.shape[:-1]
             )
             self.covariance_matrix = covariance_matrix.expand(batch_shape + (-1, -1))
-        else:
+        elif precision_matrix is not None:
```

Comment:
Legitimate fix, but should not be part of type annotation change

Reply:
This change is only necessary to make `mypy` happy once the arguments of `__init__` are annotated, it is not a bug fix.

See the notes in my OP, especially footnote 1.

Reply:
I changed it to adding a single `assert precision_matrix is not None` as in the other thread.